src/ontology/oba.Makefile

```diff
@@ -139,14 +160,24 @@ $(REPORTDIR)/%.tsv: ../sparql/synonyms-exact.sparql $(TMPDIR)/%.owl
 prepare_oba_alignment: $(REPORTDIR)/uberon.tsv $(REPORTDIR)/efo.tsv $(REPORTDIR)/pato.tsv $(REPORTDIR)/oba.tsv $(REPORTDIR)/vt.tsv $(REPORTDIR)/cl.tsv $(REPORTDIR)/go.tsv $(REPORTDIR)/chebi.tsv
 	echo "OK cool all tables prepared."
 
-CHECK_SPARQL=oba.owl
+#This is the OBA-VT DIFF Pipeline
+
+$(TMPDIR)/%.owl:
+	wget $(OBOBASE)/$*.owl -O $@
+
+$(TMPDIR)/simple-%.owl: $(TMPDIR)/%.owl data/vtoba_terms.txt 
+	$(ROBOT) merge -i $< \
+	filter -T data/vtoba_terms.txt  --select 'self annotations' --signature true -o $@
```

Comment:
```suggestion
	filter -T data/vtoba_terms.txt  --select 'self annotations children' --signature true -o $@
```

Reply:
The result still as before.

Robot command oba

```
robot --catalog catalog-v001.xml merge -i tmp/oba.owl \
filter -T data/vtoba_terms.txt  --select 'self annotations children' --signature true -o tmp/simple-oba.owl
```

Robot command vt
```
robot --catalog catalog-v001.xml merge -i tmp/vt.owl \
filter -T data/vtoba_terms.txt  --select 'self annotations children' --signature true -o tmp/simple-vt.owl
```

OAK command 
```
runoak --input sqlite:tmp/simple-oba.owl diff-via-mappings -X sqlite:tmp/simple-vt.owl --mapping-input ../mappings/oba-vt.sssom.tsv -o reports/oba-vt-diff-simple.yaml
```

Result
```
category: Identical
left_object_id: OBA:VT0100015
left_object_label: circulatory system development trait
left_predicate_id: rdfs:subClassOf
left_subject_id: OBA:VT0000121
left_subject_label: cardiovascular system development trait
right_object_id: VT:0100015
right_object_label: circulatory system development trait
right_predicate_ids:
- rdfs:subClassOf
right_subject_id: VT:0000121
right_subject_label: cardiovascular system development trait

```

Reply:
Did you check that both the ontologies have hierarchies in protege?

Reply:
Simple-VT
![simple-vt](https://user-images.githubusercontent.com/6750638/207062683-ef0cb5c0-398b-486c-873d-2298a787744b.png)
Simple-OBA
![simple-oba](https://user-images.githubusercontent.com/6750638/207062958-e563e802-06bf-4aac-b6a0-876f9ab1a270.png)

Reply:
OK seems to me that oba-simple is a flat list. Lets use a different approach. Will comment above.